### PR TITLE
query meetings through me{...}

### PR DESCRIPTION
**Pull request by cainlevy**
Fixing Pathways after https://github.com/Ameelio/connect-api-nest/pull/439.

### CODE REVIEW

File: src/redux/modules/GetMeetings.graphql [hunk 1]
```diff
@@ -1,27 +1,31 @@
...
-          firstName
-          lastName
-          profileImageUrl
-        }
-        call {
-          status
-          url
-          token
+  me {
+    ... on Inmate {
```
Comment (cainlevy) [inline src/redux/modules/GetMeetings.graphql:3]: this is a little awkward. i may follow up with an idea from last week's tour with @wittejm.

File: src/redux/modules/call.ts [hunk 2]
```diff
@@ -32,12 +32,16 @@ export const fetchCalls = createAsyncThunk(
     // GRAPHQL
     if (featureEnabled("GRAPHQL")) {
       const result = await client.query(GetMeetingsDocument).toPromise();
-      if (result.error || !result.data) {
+      if (
+        result.error ||
+        !result.data ||
+        result.data.me?.__typename !== "Inmate"
```
Comment (cainlevy) [inline src/redux/modules/call.ts:38]: also awkward. this helps typescript!

File: src/redux/modules/GetMeetings.graphql (hunk 1 above)
Comment (gabrielsaruhashi) [reply src/redux/modules/GetMeetings.graphql:3]: is this the syntax to do conditional loading?

**Review by gabrielsaruhashi — APPROVED** [review]

File: src/redux/modules/GetMeetings.graphql (hunk 1 above)
Comment (cainlevy) [reply src/redux/modules/GetMeetings.graphql:3]: yep!

we're going to get friendly with it in a timeline query.

**Pull request merged**